<launch>
    <!-- node -->
    <node pkg="roomba_500driver_meiji" exec="main500" name="main500">
        <param name="port" value="/dev/serial/roomba"/>
    </node>

    <node pkg="third_challenge" exec="third_challenge_node" name="third_challenge_node" output="screen">
        <param name="frontal_threshold" value="0.2"/>
        <param name="base_omega" value="0.6"/>
    </node>

    <node pkg="realsense2_camera" exec="realsense2_camera_node" name="realsense2_camera_node"/>

    <node pkg="ros2_yolo" exec="predictor" name="predictor"/>
</launch>